pbs_light.go

```diff
@@ -91,6 +92,15 @@ type bidResult struct {
 	bid_list pbs.PBSBidSlice
 }
 
+const DEFAULT_PRICE_GRANULARITY = "med"
```

Comment:
My thought was we'd have account settings on prebid.adnxs.com/account site. The publisher could set their price granularity there and then I could pull it in in PBS from the database. 

Since I haven't done the coding work for the prebid-site repo, I don't have any code to override this setting yet. I put a TODO to add it in the code when it is built.